front-end/src/services/userService.js

```diff
@@ -2,20 +2,12 @@ import API from 'API';
 
```

Comment:
Please follow one code style. Export single service or each function

Reply:
fix it